components/automate-gateway/pkg/nullbackend/backend.go

```diff
@@ -36,7 +36,7 @@ import (
 func NewServer() *grpc.Server {
 	// The nullbackend only listens on a unix socket. And it doesn't deal with interesting data,
 	// but only "no implemented" responses.
-	s := grpc.NewServer() // nosem
+	s := grpc.NewServer() // nosem: go.grpc.security.grpc-server-insecure-connection.grpc-server-insecure-connection
```

Comment:
In this commit, added specific rule flagged; @srenatus was there a reason you had not chose to do that for these two files?

Reply:
🤷 not really. I figured it might be better to be specific at some point and forgot to revisit the older ones.